build.gradle

```diff
@@ -58,4 +60,11 @@ jacocoTestReport {
 		xml.enabled = true
 		html.enabled = true
 	}
-}
\ No newline at end of file
+}
+
+task copyPlugins(type: Copy) {
+	from configurations.pdiPlugin
```

Comment:
BTW what reason is behind copying it into `plugins/`? what are we planning to use this folder for?